### PR TITLE
Add Azure and Google custom configs to docs

### PR DESCRIPTION
**Pull request by neoxelox**

### CODE REVIEW

File: packages/core/src/services/ai/index.ts [hunk 1]
```diff
@@ -121,8 +121,8 @@ export async function ai({
       messages,
       provider,
       apiKey,
+      url: url ?? undefined,
```
Comment (andresgutgon) [inline packages/core/src/services/ai/index.ts:124]: This is not the same as before. Make sure passing undefined for url is not having side effects

Comment (neoxelox) [reply packages/core/src/services/ai/index.ts:124]: Why it is not the same? In the function that parameter is optional

Comment (andresgutgon) [reply packages/core/src/services/ai/index.ts:124]: Now is always passing `url: undefined` before was passing nothing. Probably does not matter at all. It depneds on how is used url under the hood

Comment (neoxelox) [reply packages/core/src/services/ai/index.ts:124]: I understand that but I am calling a function with an optional parameter, it should be undefined in both cases

**Pull request merged**